README.md

```diff
@@ -275,6 +275,9 @@ MLJ provides two simple builders out of the box. In all cases weights
 
 See Table 1 above to see how `n_in` and `n_out` relate to the data.
 
+Alternatively, `MLJFlux.@builder(neural_net)` automatically creates a builder
+for any valid `neural_net`, e.g. `MLJFlux.@builder(Dense(n_in, n_out, tanh))`.
+
```

Comment:
The doc string you wrote is great, but this addition to the README could be a little a little more detailed. Maybe something along the lines of 

"Alternatively, `use MLJFlux.@builder(neural_net)` to automatically create a builder
 for any valid Flux chain expression `neural_net`, where the symbols `n_in`, `n_out`, `n_channels` and `rng` can appear literally, with the interpretations explained above. For example,

    builder = MLJFlux.@builder Chain(Dense(n_in, 128), Dense(128, n_out, tanh))
"

Reply:
Sounds good! Thanks.